python/mxnet/ndarray/numpy/_op.py

```diff
@@ -32,7 +32,8 @@
            'exp', 'expm1', 'arcsin', 'arccos', 'arctan', 'sign', 'log', 'degrees', 'log2', 'log1p',
            'rint', 'radians', 'reciprocal', 'square', 'negative', 'fix', 'ceil', 'floor',
            'trunc', 'logical_not', 'arcsinh', 'arccosh', 'arctanh', 'tensordot',
-           'linspace', 'expand_dims', 'tile', 'arange', 'split', 'concatenate', 'stack']
+           'linspace', 'expand_dims', 'tile', 'arange', 'split', 'concatenate', 'stack', 'deg2rad',
+           'rad2deg']
```

Comment:
move 'deg2rad' to after 'radians' and 'rad2deg' to after 'degrees'.

Reply:
Same for the func declarations within the file

Reply:
In all files.

Reply:
Fixed. Thank you.